testsuite/classlibrary/TestBuffer.sc

```diff
@@ -245,4 +245,36 @@ TestBuffer : UnitTest {
 		buffer.free;
 	}
 
+	test_getToFloatArray {
+		var server, data, condition, inTime = false;
+		condition = Condition.new;
+		this.bootServer;
+		server = Server.default;
```

Comment:
please do not use the default server in tests (https://github.com/supercollider/supercollider/wiki/Unit-Testing-Guide#avoid-using-the-default-server)

Reply:
OK, good point. But let's do this in a separate PR maybe? The whole file is full of references to `Server.default` ...

Reply:
You can change the other ones later, but there's no reason for new code not to be correct now